fabric-transitive-access-wideners-v1/template.accesswidener

```diff
@@ -24,4 +24,104 @@ transitive-accessible field net/minecraft/loot/LootPool bonusRolls Lnet/minecraf
+
+# Recipe factories
+transitive-accessible class net/minecraft/recipe/CookingRecipeSerializer$RecipeFactory
+transitive-accessible class net/minecraft/recipe/CuttingRecipe$Serializer$RecipeFactory
+
+# Registering custom brewing recipes
+transitive-accessible method net/minecraft/recipe/BrewingRecipeRegistry registerItemRecipe (Lnet/minecraft/item/Item;Lnet/minecraft/item/Item;Lnet/minecraft/item/Item;)V
+transitive-accessible method net/minecraft/recipe/BrewingRecipeRegistry registerPotionType (Lnet/minecraft/item/Item;)V
+transitive-accessible method net/minecraft/recipe/BrewingRecipeRegistry registerPotionRecipe (Lnet/minecraft/potion/Potion;Lnet/minecraft/item/Item;Lnet/minecraft/potion/Potion;)V
+
+# Registering custom scoreboard criteria
+transitive-accessible method net/minecraft/scoreboard/ScoreboardCriterion create (Ljava/lang/String;ZLnet/minecraft/scoreboard/ScoreboardCriterion$RenderType;)Lnet/minecraft/scoreboard/ScoreboardCriterion;
+transitive-accessible method net/minecraft/scoreboard/ScoreboardCriterion create (Ljava/lang/String;)Lnet/minecraft/scoreboard/ScoreboardCriterion;
+
+# Model part subclasses
+transitive-accessible class net/minecraft/client/model/ModelPart$Vertex
+transitive-accessible class net/minecraft/client/model/ModelPart$Quad
+
+# Creating custom render layers
+transitive-accessible method net/minecraft/client/render/RenderLayer of (Lnet/minecraft/client/render/RenderPhase$Shader;)Lnet/minecraft/client/render/RenderLayer$MultiPhaseParameters;
+
+# Creating custom sensor types
+transitive-accessible method net/minecraft/entity/ai/brain/sensor/SensorType <init> (Ljava/util/function/Supplier;)V
+
+# Creating custom activities
+transitive-accessible method net/minecraft/entity/ai/brain/Activity <init> (Ljava/lang/String;)V
+
+# Living entity methods
+transitive-accessible method net/minecraft/entity/LivingEntity damageArmor (Lnet/minecraft/entity/damage/DamageSource;F)V
+transitive-accessible method net/minecraft/entity/LivingEntity damageHelmet (Lnet/minecraft/entity/damage/DamageSource;F)V
+transitive-accessible method net/minecraft/entity/LivingEntity damageShield (F)V
+
+# Armor stand methods
+transitive-accessible method net/minecraft/entity/decoration/ArmorStandEntity isSlotDisabled (Lnet/minecraft/entity/EquipmentSlot;)Z
+transitive-accessible method net/minecraft/entity/decoration/ArmorStandEntity equip (Lnet/minecraft/entity/player/PlayerEntity;Lnet/minecraft/entity/EquipmentSlot;Lnet/minecraft/item/ItemStack;Lnet/minecraft/util/Hand;)Z
+transitive-accessible method net/minecraft/entity/decoration/ArmorStandEntity setSmall (Z)V
+transitive-accessible method net/minecraft/entity/decoration/ArmorStandEntity setShowArms (Z)V
+transitive-accessible method net/minecraft/entity/decoration/ArmorStandEntity setHideBasePlate (Z)V
+transitive-accessible method net/minecraft/entity/decoration/ArmorStandEntity setMarker (Z)V
```

Comment:
Why is this useful?

Reply:
I suppose it's only really useful to a select crowd (mostly server side devs), so I've removed it again.